pulp_2_tests/constants.py

```diff
@@ -130,6 +130,26 @@
 .. _modules.yaml: https://github.com/PulpQE/pulp-fixtures/blob/master/rpm/assets/modules.yaml
 """
 
+MODULE_FIXTURES_ERRATA = MappingProxyType({
+    'errata_id': 'RHEA-2012:0059',
+    'rpm_count': 2,
+    'total_available_units': 3,
+})
+"""The errata information containing the modular rpm data."""
+
+MODULE_ERRATA_RPM_DATA = MappingProxyType({
+    'arch': 'x86_64',
+    'collection_name': 'collection 0',
+    'context': 'deadbeef',
+    'description': 'Walrus Erratum; metadata-only',
+    'from': 'betelgeuze',
+    'issued': '2002-02-20 00:22:00',
+    'rpm_name': 'walrus',
+    'stream_name': '5.21',
+    'updated': '2018-09-05 11:23:42',
+    'version': '20180704144203',
+})
+
```

Comment:
Perhaps, add a description about this constant, or even rename it. With modularity, and modules being added this be misleading. We will need to make a clear distinction about what data we are mentioning. This constant is more related to the errata inside the `updateinfo.xml`.